### DISCUSSION TITLE
Error al hacer "git push"

### DISCUSSION THREAD

**Discussion by LuchoGM** [INGCOM-UNRN-P1/cursada-2023#18] [Preguntas y respuestas] [answered]
Venia trabajando, ya haciendo varios push y esta vez quise hacer otro y me salió este error 😭 

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/105023570/df9b5a04-2c08-4790-b7bd-c85d2d1ab5ff)

**Comment by martinvilu** [comment]
Falta cargar la llave ssh en github

**Reply by LuchoGM** [reply]
Se refiere a esto?

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/105023570/3725ab49-e7ed-4efb-afe1-c37de8856f65)

ya había hecho como 5 pushes antes de que me suceda el error

**Reply by martinvilu** [reply]
Esto esta bien, fijate con lo que dejo en el otro comentario.

**Comment by martinvilu** [comment, marked as answer]
Puede ser también que, por alguna razón, la llave salió del llavero en tu computadora (o que el programa de llavero no esté funcionando)

Proba primero ejecutar

```sh
eval `ssh-agent -s`
```

Y si eso no funciona, volve a agregarlo al llavero con 

```sh
ssh-add ~/.ssh/id_rsa
```

**Reply by LuchoGM** [reply]
Solucionado, gracias profe! 😄
